content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -263,32 +263,29 @@ The initial imported plugins file can be
 
 ### Support file
 
-By default Cypress will automatically include type-specific support files. For
-E2E, the default is `cypress/support/e2e.{js,jsx,ts,tsx}`, and for Component
-Testing `cypress/support/e2e.{js,jsx,ts,tsx}`.
-
-::testing-type-specific-option{option=supportFile}
-
-The initial imported support file can be configured to another file or turned
-off completely using the
-[supportFile](/guides/references/configuration#Folders-Files) configuration.
+We automatically seed an example support file for each configured testing type,
+which has several commented out examples.
 
 This file runs **before** every single spec file. We do this purely as a
 convenience mechanism so you don't have to import this file in every single one
 of your spec files.
 
-::testing-type-specific-option{file=partials/support-file-test-type.md}
-
 The support file is a great place to put reusable behavior such as
 [custom commands](/api/cypress-api/custom-commands) or global overrides that you
 want applied and available to all of your spec files.
 
+By default Cypress will automatically include type-specific support files. For
```

Comment:
@cowboy The supportFile section feels a little busy now. Should we consider removing this paragraph about defaults since we have the listed out files above?

Reply:
Yeah, you can remove this paragraph. Also, maybe move thee `testing-type-specific-option` directive up near the files list above.